### PR TITLE
Adds file format description

### PR DESCRIPTION
**Pull request by raabuchanan**
Adds file format description to README with input from author.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,31 @@ Dataset
 
 The generation of `data.hdf5` is specified in `gen_fb_data.py`, which requires interpolated stamped IMU measurement files and time-aligned VIO states files. The user can generate his/her own dataset with a different procedure to obtain the same fields to be used for network training and filter inputs.
 
+## File formats
+
+### Dataset lists
+
+`test.txt`, `train.txt` and `val.txt` are all the same. Each row is the name of a sequence to be used for one of the purposes of test, train or validate. The name should be the same of the sequence directory for example `seq1` and `seq22` as above.
```
Comment (CathIAS) [inline README.md:52]:
```suggestion
`test.txt`, `train.txt` and `val.txt` are list files specifying the split with one sequence name per row in the testing, training and validation datasets respectively. The name should be the same of the sequence directory for example `seq1` and `seq22` as above.
```

File: README.md [hunk 2]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+
+
+### Used to generate `data.hdf5`
+Timestamps (t) are in microseconds (us). Each row corresponds to a single timestamp. All data is delimited by commas.
```
Comment (CathIAS) [inline README.md:56]:
```suggestion
Timestamps (t) are in microseconds (us). Each row corresponds to data in a single timestamp. All data is delimited by commas.
```

File: README.md [hunk 3]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+
+- `my_timestamps_p.txt` VIO timestamps. Used to select data time range.
```
Comment (CathIAS) [inline README.md:58]:
```suggestion
- `my_timestamps_p.txt` VIO timestamps.
```

File: README.md [hunk 4]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+  - [ t ]
```
Comment (CathIAS) [inline README.md:59]:
```suggestion
  - [t]
```

File: README.md [hunk 5]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+  - Note: single column, skipped first 20 frames
+- `imu_measurements.txt` IMU data, raw and calibrated using VIO calibration
```
Comment (CathIAS) [inline README.md:61]:
```suggestion
- `imu_measurements.txt` raw and calibrated IMU data
```

File: README.md [hunk 6]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+  - [t, acc_raw (3), acc_cal (3), gyr_raw (3), gyr_cal (3), has_vio] 
+  - Note: has been interpolated evenly between images. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file (has_vio==1)
```
Comment (CathIAS) [inline README.md:63]:
```suggestion
 - Note: calibration through VIO calibration states. The data has been interpolated evenly between images around 1000Hz. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file (has_vio==1). 
```

File: README.md [hunk 7]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+- `evolving_state.txt` VIO evolving states at IMU rate.
```
Comment (CathIAS) [inline README.md:64]:
```suggestion
- `evolving_state.txt` ground truth (VIO) states at IMU rate.
```

File: README.md [hunk 8]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+  - [t, q_wxyz (4), p (3), v (3)]
+  - Note: not interpolated, raw IMU timestamps. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file
```
Comment (CathIAS) [inline README.md:66]:
```suggestion
  - Note: VIO state estimates with IMU integration. Timestamps are from raw IMU measurements.
```

File: README.md [hunk 9]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+- `calib_state.txt` VIO calibration states at image rate (used in data_io.py)
```
Comment (CathIAS) [inline README.md:67]:
```suggestion
- `calib_state.txt` VIO calibration states at image rate (used in `data_io.py`)
```

File: README.md [hunk 10]
```diff
@@ -45,6 +45,31 @@ Dataset
...
+  - [t, acc_scale_inv (9), gyr_scale_inv (9), gyro_g_sense (9), b_acc (3), b_gyr (3)]
+  - Note: Changing calibration states from VIO.
+- `atttitude.txt` AHRS attitude from IMU
+  - [ t, qw, qx, qy, qz ]
```
Comment (CathIAS) [inline README.md:71]:
```suggestion
  - [t, qw, qx, qy, qz]
```

**Review by CathIAS — COMMENTED** [review]
Looks good! Made some suggestions to be more precise.

**Comment by raabuchanan** [comment]
Ok, I accepted all the changes. Shall I merge?

**Pull request merged**